### DISCUSSION TITLE
Force dark mode

### DISCUSSION THREAD

**Discussion by jcubic** [mantinedev/mantine#7308] [Q&A] [answered]
Is there a way to force dark mode with SSR? I don't want to switch between modes. I want to force dark mode. The only way I see to change the mode is to use provider, and it's client code.

**Comment by rtivital** [comment, marked as answer]
https://mantine.dev/theming/color-schemes/#force-color-scheme

**Reply by jcubic** [reply]
Thanks, this should be somehow linked from first code snippet.

```
import { MantineProvider } from '@mantine/core';

function Demo() {
  return (
    <MantineProvider defaultColorScheme="dark">
      {/* Your app here */}
    </MantineProvider>
  );
}
```

I assumed that this is the way you change theme, and you don't need to enforce it.
